src/miner.cpp

```diff
@@ -177,7 +180,44 @@ bool BlockAssembler::isStillDependent(CTxMemPool::txiter iter)
     return false;
 }
 
+void BlockAssembler::onlyUnconfirmed(CTxMemPool::setEntries& testSet)
+{
+    for (CTxMemPool::setEntries::iterator iit = testSet.begin(); iit != testSet.end(); ) {
+        // Only test txs not already in the block
+        if (inBlock.count(*iit)) {
+            testSet.erase(iit++);
+        }
+        else {
+            iit++;
+        }
+    }
+}
```

Comment:
Nit: may as well have ++iit in the for loop because it always occurs.

```
for (CTxMemPool::setEntries::iterator iit = testSet.begin(); iit != testSet.end(); ++iit) {
         // Only test txs not already in the block
         if (inBlock.count(*iit)) 
             testSet.erase(iit);
         
     }
 }
```